### PR TITLE
First pass at forcats 1.0.0 blog post

### PR DESCRIPTION
**Pull request by hadley**

### CODE REVIEW

File: content/blog/forcats-1-0-0/index.Rmd [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+
+We're so happy to announce the release of [forcats](https://forcats.tidyverse.org) 1.0.0. The goal of the forcats package is to provide a suite of tools that solve common problems with factors, including changing the order of levels or the values.
+
+You can install it from CRAN with:
+
+```{r, eval = FALSE}
+install.packages("forcats")
+```
+
+While this is the 1.0.0 release of forcats, this is mainly a signal that we think forcats is stable, and don't anticipated any major changes in the future. This blog post will outline the only major new features in this version: `fct_na_level_to_value()` and `fct_na_value_to_level()`. As usual, you can see a full list of changes in the [release notes](https://github.com/tidyverse/forcats/releases/tag/v1.0.0).
```
Comment (mine-cetinkaya-rundel) [inline content/blog/forcats-1-0-0/index.Rmd:42]:
```suggestion
While this is the 1.0.0 release of forcats, this is mainly a signal that we think forcats is stable, and don't anticipate any major changes in the future. This blog post will outline the only major new features in this version: `fct_na_level_to_value()` and `fct_na_value_to_level()`. As usual, you can see a full list of changes in the [release notes](https://github.com/tidyverse/forcats/releases/tag/v1.0.0).
```

File: content/blog/forcats-1-0-0/index.Rmd [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+    This requires a little more work to create, because, by default, `factor()` uses `exclude = NA`, meaning that missing values are not included in the levels.
+    You can force `NA` to be included by setting `exclude = NULL`:
+
+    ```{r}
+    f2 <- factor(c("x", "y", NA), exclude = NULL)
+    levels(f2)
+    is.na(f2)
+    ```
+
+You can see the difference a little more clearly by looking at the underlying integer value of the factor:
```
Comment (mine-cetinkaya-rundel) [inline content/blog/forcats-1-0-0/index.Rmd:71]: integer values (plural)?

File: content/blog/forcats-1-0-0/index.Rmd [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+      fct_na_value_to_level() |> 
+      fct_infreq() |> 
+      fct_rev()
+  ) |> 
+  ggplot(aes(y = hair_color)) + 
+  geom_bar() + 
+  labs(y = "Hair color")
+```
+
+Pulling the factor manipulation out into it's own mini-pipeline makes it easier to make other adjustments. For example, the code below uses a more informative label for the missing level, and lumps together all the hair colours with less than 2 observations. I've left the (Other) category as a bar at the end, but by rearranging the other of `fct_infreq()` and `fct_lump_min()` I could other with the others, if I wanted.
```
Comment (mine-cetinkaya-rundel) [inline content/blog/forcats-1-0-0/index.Rmd:137]:
```suggestion
Pulling the factor manipulation out into its own mini-pipeline makes it easier to make other adjustments. For example, the code below uses a more informative label for the missing level and lumps together all the hair colours with less than 2 observations. I've left the (Other) category as a bar at the end, but by rearranging the other of `fct_infreq()` and `fct_lump_min()` I could other with the others, if I wanted.
```

Comment (mine-cetinkaya-rundel) [reply content/blog/forcats-1-0-0/index.Rmd:137]: Some words are missing here I think? could "lump" with the others maybe?

File: content/blog/forcats-1-0-0/index.Rmd [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+We're so happy to announce the release of [forcats](https://forcats.tidyverse.org) 1.0.0.
+The goal of the forcats package is to provide a suite of tools that solve common problems with factors, including changing the order of levels or the values.
+
+You can install it from CRAN with:
+
+```{r, eval = FALSE}
+install.packages("forcats")
+```
+
+While this is the 1.0.0 release of forcats, this version number is mainly a signal that we think forcats is stable, and we don't anticipate any major changes in the future.
```
Comment (batpigandme) [inline content/blog/forcats-1-0-0/index.Rmd:44]:
```suggestion
While this is the 1.0.0 release of forcats, this version number is mainly to signal that we think forcats is stable, and that we don't anticipate any major changes in the future.
```

**Review by batpigandme — APPROVED** [review]
Just one little change.

**Pull request merged**